docs/live-migration.md

```diff
@@ -0,0 +1,125 @@
+Live migration is the act of moving a virtual machine from one
+hypervisor to another while maintaining the connectivity and liveness
+of the applications running on the virtual machine.  It is one of the
+primary advantages of using an external VM management service.
+
+Triggering a live migration requires that the user perform an API
+call.
+
+To Create (start migration):
+
+`kubectl create -f migration.yaml`
+
+Here is an example of a  migration request in YAML format:
+
+```yaml
+
+apiVersion: kubevirt.io/v1alpha1
+kind: Migration
+metadata:
+  name: testvm-migration
+spec:
+  selector:
+    name: testvm
+```
+
+The selector section indicates which vm objects are to be
+migrated. The name field shown here would match a vm with the `name`
+value of `testvm.`
+
+To list all migrations:
+
+`kubectl get migrations`
+
+To query a specific migration:
+
+`kubectl get migrations <migration_name>`
```

Comment:
I think the user would need to specify `kubectl get migrations <migration_name> -o yaml` to get the output shown.